_posts/2018-03-25-nlog-4-5-is-live.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: NLog 4.5 is live!
+---
+
+NLog 4.5 is now finally ready after 2 years of waiting for NLog 5.0 BETA to be ready. NLog 5.0 BETA has now been abandoned,
+and people should instead use NLog 4.5.
+
+## NetCore Support
+
+NLog 4.5 provides official support for NetCore without any breaking changes!
+
+- NetStandard 1.3 - Minimal dependencies for limited NetCore1 platforms (Ex. UWP1)
```

Comment:
FYI, I think you're the only person who is using NetStandard / NetCore as aliases for .NET Standard / .NET Core ;)

(I've changed it after this merge)

Reply:
I'm just camel-casing the defines used by the compiler :)